packages/abnormal_security/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.0.0"
+  changes:
+    - description: Migrate to GA.
```

```suggestion
    - description: Release package as GA.
```
? (others too if the change is made here)

---

Updated text